atmos/modules/terraform/terraform-console.variant

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env variant
+# vim: filetype=hcl
+
+job "terraform console" {
+  concurrency = 1
+  description = "Run 'terraform console'"
+
+  parameter "component" {
+    type        = string
+    description = "Component"
+  }
+
+  option "command" {
+    default     = "terraform"
+    type        = string
+    description = "Command to execute, e.g. 'terraform', or path to the command, e.g. '/usr/local/terraform/0.13/bin/terraform'"
+  }
+
+  option "stack" {
+    type        = string
+    description = "Stack"
+    short       = "s"
+  }
+
+  option "args" {
+    default     = ""
+    description = "A string of arguments to supply to the terraform command"
+    type        = string
+  }
+
+  option "interactive" {
+    type        = bool
+    description = "Interactive"
+    short       = "i"
+    default     = false
```

Comment:
I am not positive what "Interactive" does or if it is even implemented, but `console` should be interactive by default
```suggestion
    default     = true
```